packages/components/src/Account/Auth/EthAuth.tsx

```diff
@@ -113,10 +153,24 @@ export class AccountEthAuth extends React.Component<AccountEthAuthProps, Account
         },
         postTransaction: async (sig: EthSignedMessage): Promise<void> => {
           try {
-            await userSetEthAddress(sig);
-            this.setState({ isWaitingSignatureOpen: false });
-            if (this.props.onAuthenticated) {
-              this.props.onAuthenticated(sig.signer);
+            delete sig.rawMessage; // gql endpoint doesn't want this and errors out
```

Comment:
why delete instead of `= null`;

Reply:
`sig` is of type `EthSignedMessage` which expects `rawMessage` to be a string, so setting it to undefined or null is a type error, but (for some reason) deleting it doesn't. Other option is to typecast in a couple places or tslint ignore that line. Any reason not to use `delete`?